src/Microsoft.VisualBasic/ref/project.json

```diff
@@ -1,12 +1,8 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0"
+    "System.Runtime": "4.1.0"
```

Comment:
Sorry I miss read your description thinking you were going to reference the latest version of all packages, i.e. prerelease versions.